x-pack/plugins/maps/public/connected_components/toolbar_overlay/feature_draw_controls/feature_edit_tools/feature_edit_tools.tsx

```diff
@@ -36,6 +36,18 @@ export function FeatureEditTools(props: Props) {
   const drawPointSelected = props.drawShape === DRAW_SHAPE.POINT;
   const deleteSelected = props.drawShape === DRAW_SHAPE.DELETE;
 
+  const [drawMode, setDrawMode] = useState('');
```

Comment:
I'm not sure if we need to introduce local state. This is equivalent to `props.drawShape`.